generate/graphql.go

```diff
@@ -200,9 +200,31 @@ func buildGraphQLFile(schemaPath, name, packageName string, args BuildArgs) erro
 			generated.rootStruct.fields[lf] = lfd[lf]
 			generated.rootStruct.fieldOrder = append(generated.rootStruct.fieldOrder, lf)
 		}
+
+		gfile := gfile
```

Comment:
As a potential nit, should this be `gFile`?

Reply:
Strictly speaking, yes, however that's the name of the variable from the existing (`generate.go`) code so I kept it like that.